yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```

Comment:
What are these files doing here? 

Could you try deleting this file and the `node_modules` folder 

Then run the project again with `make` to see if they reappear

I think they are here because you have run `yarn install` while debugging.

Let me know what happens. We may need to update the `gitignore` file so these files are not tracked.

Reply:
I removed this and they don't reappear when running make.